cmd/helm/registry_logout.go

```diff
@@ -35,7 +35,7 @@ func newRegistryLogoutCmd(cfg *action.Configuration, out io.Writer) *cobra.Comma
 		Short:             "logout from a registry",
 		Long:              registryLogoutDesc,
 		Args:              require.MinimumNArgs(1),
-		ValidArgsFunction: noCompletions,
+		ValidArgsFunction: cobra.NoFileCompletions,
```

Comment:
This turns off file completion but does not print the ActiveHelp message since an argument is expected, it is just that it should not be a file. I'm plan to improve on this with a specific ActiveHelp message in a future PR.